simulation/halsim_ds_socket/build.gradle

```diff
@@ -0,0 +1,27 @@
+description = "A plugin that listens on a socket so that you can use the real Driver Station software to connect to the simulation"
+
+ext {
+    includeWpiutil = true
+    pluginName = 'halsim_ds_socket'
+}
+
+ext.cppfrccomm = { binary->
+    binary.tasks.withType(CppCompile) {
+            binary.cppCompiler.args "-I" + project(':ni-libraries').file('include').absolutePath
```

Comment:
Adding `-I` arguments directly is unreliable in gradle. I think I would rather change the dependency set to allow header only linking? Also is the only reason netcomm is needed is for the structs? If thats the case, I would rather just recreate them, rather then pulling in the netcomm headers.